src/chart/Treemap.tsx

```diff
@@ -118,7 +116,7 @@ export const computeNode = ({
     nodeValue = computedChildren.reduce((result: any, child: TreemapNode) => result + child[NODE_VALUE_KEY], 0);
   } else {
     // TODO need to verify dataKey
-    nodeValue = isNan(node[dataKey as string]) || node[dataKey as string] <= 0 ? 0 : node[dataKey as string];
+    nodeValue = Number.isNaN(node[dataKey as string]) || node[dataKey as string] <= 0 ? 0 : node[dataKey as string];
```

Comment:
lodash `isNan` converts values to a number and returns true if this can be the case 

Number.isNaN does not. We might want to use es-toolkit for this for parity's sake

Reply:
Looks like es-toolkit uses `Number.isNaN()`. I'll raise an issue on their repo and see if they can change that. For now I'll just create a function within `DataUtils`.

[Lodash isNan](https://github.com/lodash/lodash/blob/f299b52f39486275a9e6483b60a410e06520c538/lodash.js#L11960)
[es-toolkit isNan](https://github.com/toss/es-toolkit/blob/a46d8c4969afa5930bccfea07c14752b7f6be0d3/src/compat/predicate/isNaN.ts)

Reply:
I doubt they'll change it. But here I'm mostly concerned that we rely on the conversion in the case that someone provides data that is quoted numbers or something

Reply:
I added some tests, let me know if I need to add any more. I also changed `isNan` with `Number.isNaN` in the tests and it all worked as expected.

Also, double checked and made sure that lodash `isNaN` returned the same results as the isNan function.